pallet/src/tests/signer.rs

```diff
@@ -497,3 +583,71 @@ fn verify_old_multi_signer_requests_getting_removed() {
             );
         })
 }
+
+#[test]
+fn cheque_limit_in_multi_signer_execution_works() {
```

Comment:
if my assumption about the test is true, then maybe:
```suggestion
fn insufficient_cheque_limit_aborts_the_multisig_script_works() {
```

Reply:
Done